### PR TITLE
Fix multiple user import error message

### PR DESCRIPTION
**Pull request by PasinduYeshan**
### Purpose
- Change multiple user import error message when username doesnt match the regex pattern
- Remove the unwanted helper panel info
image

- Change parent user invite hint
image

- Make email domain discovery organization name input box read only
image

### Related Issues
- https://github.com/wso2/product-is/issues/18942

### Related PRs
- Related PR `#1` or (None)

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: modules/i18n/src/translations/en-US/portals/console.ts [hunk 1]
```diff
@@ -9461,12 +9461,12 @@ export const console: ConsoleNS = {
...
                 addUserWizard: {
                     heading: "Invite Parent User",
-                    description: "Invite a user from the parent organization.",
+                    description: "Invite users from the parent organization.",
                     hint: "Invited users are managed by the parent organization.",
                     username: {
                         label: "Username",
                         placeholder: "Enter the username",
-                        hint: "Username should belong to a user from the parent organization.",
+                        hint: "Add the username of a parent user and press enter. Repeat to include multiple users.",
```
Comment (dasuni-30) [inline modules/i18n/src/translations/en-US/portals/console.ts:9469]: Check with the Doc team if we can improve this content

**Review by dasuni-30 — APPROVED** [review]

File: modules/i18n/src/translations/en-US/portals/console.ts (hunk 1 above)
Comment (PasinduYeshan) [reply modules/i18n/src/translations/en-US/portals/console.ts:9469]: Approved by doc team

File: apps/console/src/features/users/components/wizard/bulk-import-user-wizard.tsx [hunk 2]
```diff
@@ -159,6 +159,7 @@ const ADDRESS_ATTRIBUTE: string = "addresses";
 const HOME_ATTRIBUTE: string = "home";
 const BULK_ID: string = "bulkId";
 const FILE_IMPORT_TIMEOUT: number = 60000; // 1 minutes.
+const usernameRegexErrorCode: string = "31301";
```
Comment (pavinduLakshan) [inline apps/console/src/features/users/components/wizard/bulk-import-user-wizard.tsx:162]: Shall we move this to the constants [1] folder?

[1] https://github.com/wso2/identity-apps/blob/master/apps/console/src/features/users/constants/user-management-constants.ts

File: apps/console/src/features/organization-discovery/components/edit-organization-discovery-domains.tsx [hunk 3]
```diff
@@ -275,9 +275,7 @@ const EditOrganizationDiscoveryDomains: FunctionComponent<EditOrganizationDiscov
                                     </Hint>
                                 ) }
                                 component={ TextFieldAdapter }
-                                InputProps={ {
-                                    readOnly: true
-                                } }
+                                readOnly
```
Comment (pavinduLakshan) [inline apps/console/src/features/organization-discovery/components/edit-organization-discovery-domains.tsx:278]: Shouldn't we make this true/false based on the scopes/roles?

Comment (PasinduYeshan) [reply apps/console/src/features/organization-discovery/components/edit-organization-discovery-domains.tsx:278]: This organization name shouldn't be changed.

File: apps/console/src/features/users/components/wizard/bulk-import-user-wizard.tsx (hunk 2 above)
Comment (PasinduYeshan) [reply apps/console/src/features/users/components/wizard/bulk-import-user-wizard.tsx]: Moved

**Review by pavinduLakshan — APPROVED** [review]

**Pull request merged**